packages/block-library/src/social-link/index.php

```diff
@@ -19,14 +19,15 @@ function render_block_core_social_link( $attributes ) {
 		$attributes['label'] :
 		/* translators: %s: Social Link service name */
 		sprintf( __( 'Link to %s' ), block_core_social_link_get_name( $service ) );
+	$class_name = ( isset( $attributes['className'] ) ) ? ' ' . $attributes['className'] : false;
```

Comment:
It seems we can remove the external parentheses `( isset( $attributes['className'] ) )` right?

Reply:
Yeah no problem, that was just how the rest of the ternary assignments were written for the other attributes in the function so I thought it might be a WordPress coding convention, I can update

Reply:
Updated

Reply:
Sorry did not notice the other attributes also had parentheses, anyway I think they are not required so thank you for the update 👍